### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/1141e3e8a5149b918f188d0c4d63ce7a97c78bc8

### CODE REVIEW

File: configs/pontoon.toml [hunk 1]
```diff
@@ -189,9 +189,6 @@ locales = [
 [[paths]]
     reference = "en/mozorg/about/history.ftl"
     l10n = "{locale}/mozorg/about/history.ftl"
-[[paths]]
```
Comment (alexgibson) [inline configs/pontoon.toml:192]: Note: This should not be removed

File: en/privacy/faq.ftl [hunk 2]
```diff
@@ -45,7 +47,7 @@ privacy-faq-we-make-our-documentation-public = We make our documentation public
 privacy-faq-that-documentation-is-gobbledygook = That documentation is gobbledygook to me! Can you give it to me in plain English?
 privacy-faq-there-are-two-categories-of = There are two categories of data that we collect by default in our release version of { -brand-name-firefox }.
 privacy-faq-the-first-is-what-we-call-technical = The first is what we call “technical data.” This is data about the browser itself, such as the operating system it is running on and information about errors or crashes.
-privacy-faq-the-second-is-what-we-call-interaction = The second is what we call “interaction data.” This is data about an individual’s engagement with { -brand-name-firefox }, such as the number of tabs that were open, the status of user preferences, or number of times certain browser features were used, such as screenshots or containers. For example, we collect this data in terms of the back button, that arrow in the upper left corner of your browser that lets you navigate back to a previous webpage in a way that shows us someone used the back button, but doesn’t tell what specific webpages are accessed.
```
Comment (alexgibson) [inline en/privacy/faq.ftl:48]: @craigcook looks like this is reverting some changes that were made in this repo, but not in the bedrock repo? Do we need to get things in sync?

Comment (craigcook) [reply en/privacy/faq.ftl:48]: Yeah, this string should be in bedrock as well. Sorry I missed it.

**Review by peiying2 — APPROVED** [review]
